frontend/tests/components/ZipDownloadButton.test.tsx

```diff
@@ -0,0 +1,17 @@
+import { render, screen } from "tests/react-utils";
+
+import ZipDownloadButton from "src/components/opportunity/ZipDownloadButton";
+
+const ZipDownloadButtonProps = {
+  opportunityId: 87,
+  testId: `opportunity-document-button`,
+};
+
+describe("ZipDownloadButton", () => {
+  it("Renders without errors", () => {
+    render(<ZipDownloadButton {...ZipDownloadButtonProps} />);
+    const zipDownloadButton = screen.getByTestId(`opportunity-document-button`);
```

Comment:
passing in the test id probably isn't necessary. I bet you can get what you want with `screen.getByRole('button')` since there's only one button being rendered